pages/common/id3tag.md

```diff
@@ -5,7 +5,7 @@
 
 - Set artist and title tag of an MP3 file:
 
-`id3tag --artist={{artist}} --title={{title}} {{path/to/file.mp3}}`
+`id3tag --artist={{artist}} --song={{title}} {{path/to/file.mp3}}`
```

Comment:
```suggestion
`id3tag --artist {{artist}} --song {{title}} {{path/to/file.mp3}}`
```

Reply:
The command itself seems to support the spaced syntax, but its help text explicitly mentions the `=` form:

```plaintext
id3tag 
Usage: id3tag [OPTIONS]... [FILES]...
   -h         --help            Print help and exit
   -V         --version         Print version and exit
   -1         --v1tag           Render only the id3v1 tag (default=off)
   -2         --v2tag           Render only the id3v2 tag (default=off)
   -aSTRING   --artist=STRING   Set the artist information
   -ASTRING   --album=STRING    Set the album title information
   -sSTRING   --song=STRING     Set the title information
   -cSTRING   --comment=STRING  Set the comment information
   -CSTRING   --desc=STRING     Set the comment description
   -ySTRING   --year=STRING     Set the year
   -tSTRING   --track=STRING    Set the track number
   -TSTRING   --total=STRING    Set the total number of tracks
   -gSHORT    --genre=SHORT     Set the genre
   -w         --warning         Turn on warnings (for debugging) (default=off)
   -n         --notice          Turn on notices (for debugging) (default=off)
```

I'd rather stay true to it, but let me know what you prefer.

Reply:
Except for special cases (e.g. when the command doesn't support it or maybe for clarity), I think it is better to follow the style guide. In my opinion, it's better to be consistent with our own pages than follow a pattern from the program's docs. It don't think it makes sense to keep the "=" in this case.